lib/config/wrapper.go

```diff
@@ -92,6 +93,7 @@ type Wrapper interface {
 	RawCopy() Configuration
 	RequiresRestart() bool
 	Save() error
+	Finish(w http.ResponseWriter, waiter Waiter)
```

Comment:
This is an api specific utility, i.e. shouldn't live in the config package and remain in the api package.  
Incidentlally it's also what makes CI break (and tests not run), as you'd need to run `go run build.go testmocks` to regenerate the mock struct for this interface. However not needed when just not adding it here.

Reply:
Ok, thanks, I'll see if I can refactor this.